### PR TITLE
Updated Azazel information

### PR DESCRIPTION
**Pull request by gruunday**

### CODE REVIEW

File: docs/machines/azazel.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+* Type:   Dell PowerEdge R515
+* OS: Ubuntu 14.04
+* CPU:    2 x AMD Opteron 4180 @ 2.6Ghz
+* RAM:    16GB
+* Storage:    Dell PERC H200 Integrated RAID Controller
+* Disks:  2 x 146GB 15,000 RPM SAS in RAID 1
+* DAS:    Worf
+* Drives: Internal SATA DVD+/-RW
+* Network:    2x Onboard Ethernet
```
Comment (butlerx) [inline docs/machines/azazel.md]: could you remove the tabs

File: docs/machines/azazel.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+## Where to find
+
+* 136.206.15.25
```
Comment (butlerx) [inline docs/machines/azazel.md]: this is pyg ip address

File: docs/machines/azazel.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+* 136.206.15.24
+* 136.206.15.73
+* 136.206.15.50 
+
+## Services
+
+* Serves /home and /webtree over [NFS](/services/nfs).
```
Comment (butlerx) [inline docs/machines/azazel.md]: technically it services `/storage` which which webtree and home live in

**Comment by butlerx** [comment]
could you also in future fork before you make changes

File: docs/machines/azazel.md (hunk 2 above)
Comment (zergless) [inline docs/machines/azazel.md]: no is no reverse dns for this FYI

**Pull request merged**